### PR TITLE
Changed the time billing period to last 31 days

### PR DESCRIPTION
**Pull request by athiruma**
1. Extended the TimePeriod for the last 31 days

### CODE REVIEW

File: cloud_governance/policy/aws/cost_explorer.py [hunk 1]
```diff
@@ -15,6 +16,8 @@ class CostExplorer:
     fetching AWS cost explorer of two days ago because day ago cost calculation is not closed.
     """
 
+    BULK_UPLOAD = 8
```
Comment (ebattat) [inline cloud_governance/policy/aws/cost_explorer.py:19]: BULK_UPLOAD_THREADS

File: jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py [hunk 2]
```diff
@@ -39,11 +39,10 @@
...
-os.system(f"""podman run --rm --name cloud-governance -e AWS_DEFAULT_REGION="us-east-1" -e account="perf-scale" -e policy="cost_explorer" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_index="{es_index_global}" -e cost_explorer_tags="{cost_tags}" -e granularity="{granularity}" -e cost_metric="{cost_metric}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+# os.system(f"""podman run --rm --name cloud-governance -e AWS_DEFAULT_REGION="us-east-1" -e account="perf-dept" -e policy="cost_explorer" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_index="{es_index_global}" -e cost_explorer_tags="{cost_tags}" -e granularity="{granularity}" -e cost_metric="{cost_metric}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:42]: Why did u comment it ?

File: cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py [hunk 3]
```diff
@@ -17,8 +17,8 @@ def get_cost_by_tags(self, tag: str, granularity: str = 'DAILY', cost_metric: st
         @return:
         """
         if not start_date and not end_date:
-            end_date = datetime.now() - timedelta(1)
-            start_date = end_date - timedelta(1)
+            end_date = datetime.now() + timedelta(1)
+            start_date = end_date - timedelta(31)
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py:21]: START_DAY = 1
END_DAY = 31

File: jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py (hunk 2 above)
Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:42]: No need this.
I think better to keep in comments.

Comment (ebattat) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:42]: Need to add todo, to remove if if cost explorer is working properly

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**